src/contracts/atlas/AtlETH.sol

```diff
@@ -200,11 +201,11 @@ abstract contract AtlETH is Permit69 {
         EscrowAccountAccessData storage s_aData = S_accessData[owner];
 
         s_aData.bonded -= _amt;
-        s_aData.lastAccessedBlock = uint32(block.number);
+        s_aData.lastAccessedBlock = uint32(SafeBlockNumber.get());
 
         s_balanceOf[owner].unbonding += _amt;
 
-        emit Unbond(owner, amount, block.number + ESCROW_DURATION + 1);
+        emit Unbond(owner, amount, SafeBlockNumber.get() + ESCROW_DURATION + 1);
```

Comment:
We should probably cache the block number in a local var if used more than once in a function. Looks like thats done in the other files but not here. Any reason not to here?

Reply:
I will work on a PR for the transient storage idea which and add a caches here as well